docs/en_US/Release.md

```diff
@@ -1,5 +1,31 @@
 # ChangeLog
 
+## Release 0.9 - 7/1/2019
+
+### Major Features
+* General NAS programming interface
+    * Add `enas-mode` and `oneshot-mode` for NAS interface: [PR #1201](https://github.com/microsoft/nni/pull/1201#issue-291094510)
+* [Gaussian Process Tuner with Matern kernel](./GPTuner.md) 
+
+* Multiphase interface
+    * Support for OpenPAI Training Service
+    * Enable all builtin tuners as multiphase tuner. For details, please refer to [Write a tuner that leverages multi-phase](./MultiPhase.md#write-a-tuner-that-leverages-multi-phase)
+
+* Web Portal
```

Comment:
@lvybriage - could you please add few screenshot here for the Web UI changes?

Reply:
Ok.

Reply:
@scarlett2018 added link to webUI doc: https://github.com/microsoft/nni/pull/1232/commits/030965f6e1acb517d4308188fa3ab6a7bf2e4c69